pom.xml

```diff
@@ -0,0 +1,251 @@
+                            version="${carbon.identity.framework.imp.pkg.version.range}",
+                            org.wso2.carbon.identity.application.authentication.framework.util;
+                            version="${carbon.identity.framework.imp.pkg.version.range}",
+                            org.wso2.carbon.identity.application.authentication.framework.exception;
+                            version="${carbon.identity.framework.imp.pkg.version.range}",
+                            org.wso2.carbon.identity.application.authentication.framework.model;
+                            version="${carbon.identity.framework.imp.pkg.version.range}",
+                            org.wso2.carbon.identity.application.authentication.framework.config.model.graph.js;
+                            version="${carbon.identity.framework.imp.pkg.version.range}",
+                            org.wso2.carbon.identity.application.authentication.framework.config.model.graph;
+                            version="${carbon.identity.framework.imp.pkg.version.range}",
+                            org.wso2.carbon.identity.application.authentication.framework.config.model;
+                            version="${carbon.identity.framework.imp.pkg.version.range}",
+                            org.wso2.carbon.identity.application.authentication.framework.context;
+                            version="${carbon.identity.framework.imp.pkg.version.range}",
+                            org.wso2.carbon.identity.application.common.model,
+                            org.wso2.carbon.identity.governance,
+                            org.wso2.carbon.identity.governance.common,
+                            org.wso2.carbon.identity.event,
+                            org.apache.commons.collections;version="${apache.commons.collections.imp.pkg.version}",
+                            org.wso2.carbon.registry.core.service,
+                            org.wso2.carbon.user.core;version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.user.core.claim;version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.user.core.service;version="${carbon.kernel.package.import.version.range}",
+                            org.json.simple,
+                            org.json.simple.parser,
+                            javax.servlet.http;version="${imp.pkg.version.javax.servlet}"
+                        </Import-Package>
+                    </instructions>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
+    <properties>
+        <carbon.kernel.version>5.2.13</carbon.kernel.version>
+        <carbon.utils.version>4.6.1</carbon.utils.version>
+        <carbon.user.core.version>4.6.1</carbon.user.core.version>
```

Comment:
You can use `<carbon.kernel.version>` for these two as well. No need to define this separately.